node-helpers/dev-server-sse/index.ts

```diff
@@ -30,17 +41,13 @@ if (path.isAbsolute(filePathOfGameRecording)) {
 
 main(resolvedFilePathOfGameRecording);
 
-function timeout(ms: number) {
-  return new Promise(resolve => setTimeout(resolve, ms));
-}
-
 async function* infiniteAsyncGeneratorFromArrayString(input: Array<string>, delay = 120) {
   let currentIndex = 0;
   yield input[currentIndex];
   while (true) {
     currentIndex++;
     if (input[currentIndex]?.trim()) {
-      await timeout(120);
+      await setTimeout(delay);
```

Comment:
J'avais raté ça.

Reply:
`timersPromises.setTimeout([delay[, value[, options]]])` https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options

`value` is optional, eslint doesn't identify it's not the regular `setTimeout` ?

Reply:
I meant you wouldn't have missed that you were not using `delay` using this rule.

Reply:
ok, for the moment, I haven't setup any eslint for this part.